cps/main.c

```diff
@@ -592,30 +588,47 @@ assign_cps_queue_ids(struct cps_config *cps_conf)
 	 */
 
 	if (cps_conf->net->front.rss) {
+		cps_conf->mp = create_pktmbuf_pool("cps",
+			cps_conf->lcore_id, num_mbuf);
+		if (cps_conf->mp == NULL) {
+			ret = -1;
+			goto fail;
+		}
+
```

Comment:
The mbuf pool of CPS is not optional because of the calls to `rte_pktmbuf_alloc()` in its code. But the size of the pool may vary depending on hardware support.